test/e2e_node/cpu_manager_test.go

```diff
@@ -181,6 +182,21 @@ func enableCPUManagerInKubelet(f *framework.Framework) (oldCfg *kubeletconfig.Ku
 	return oldCfg
 }
 
+// wait for local Kubelet to be healthy
+func waitForKubeletUp() error {
```

Comment:
Probably reuse https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/cpu_manager_test.go#L175-L179 and remove this?

Reply:
Api-server has timeout to report whether node is up/down (default 5s), checking for `healthz` seems more accurate since we know that when `ok 200` is posted, cpu manager has started with reconcile loop (`cpumanager.Start()` has been called)

Reply:
Ok. We can reuse https://github.com/kubernetes/kubernetes/blob/master/test/e2e/framework/util.go#L3832?

Reply:
I think it is pointless to ssh to localhost only to do curl.

Reply:
This method SSH to localhost in order to execute curl, are all e2e k8s environments configured to support such options.